modules/web/src/plugins/ballerina/utils/react-try-catch-batching-strategy.js

```diff
@@ -1,50 +0,0 @@
-/**
```

Comment:
Shall we add error boundaries to ballerina file editor and diagram.

Reply:
@hevayo pushed Error Boundaries component
![selection_096](https://user-images.githubusercontent.com/1552869/34556482-75348850-f15c-11e7-97b3-7e7e1b3b0338.png)